server/src/main/java/io/littlehorse/common/model/getable/global/wfspec/variable/VariableMutationModel.java

```diff
@@ -155,7 +155,7 @@ public void execute(ThreadRunModel thread, Map<String, VariableValueModel> txnCa
         } catch (LHVarSubError exn) {
             throw exn;
         } catch (Exception exn) {
-            log.error(exn.getMessage(), exn);
+            log.trace(exn.getMessage(), exn);
```

Comment:
Keep this?

Reply:
This is a user-level error that will show up in the `errorMessage` on the `NodeRun`. It's not an actual exception but rather the User's WfSpec is invalid. We don't want that polluting our stacktraces—they should debug it using the dashboard or `lhctl` rather than looking at our server logs (especially because in LH Cloud they won't see the logs).